tests/pr_test.go

```diff
@@ -59,6 +60,20 @@ func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptio
 			Prefix:        prefix,
 			ResourceGroup: resourceGroup,
 			Region:        region,
+			TerraformVars: map[string]interface{}{
+				"access_tags": permanentResources["accessTags"],
+			},
+		}) // fsCloudTerraformDir does not implement any access tag functionality
+	} else if dir == fsCloudTerraformDir {
+		options = testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+			Testing:       t,
+			TerraformDir:  dir,
+			Prefix:        prefix,
+			ResourceGroup: resourceGroup,
+			Region:        region,
+			TerraformVars: map[string]interface{}{
+				"existing_at_instance_crn": permanentResources["activityTrackerFrankfurtCrn"],
+			},
```

Comment:
access tags should be exposed as a variable in the `profiles)/fscloud` submodule. then you wont need to have this extra conditional logic as both examples will have `access_tags` variable